api/src/main/java/eu/europeana/fulltext/api/web/FTWriteController.java

```diff
@@ -370,6 +370,7 @@ protected ResponseEntity<String> generateResponse(
   private ResponseEntity<String> noContentResponse(HttpServletRequest request) {
     org.springframework.http.HttpHeaders headers = new org.springframework.http.HttpHeaders();
     headers.add(HttpHeaders.ALLOW, getMethodsForRequestPattern(request, requestPathMethodService));
+    headers.add(HttpHeaders.CONTENT_TYPE, HttpHeaders.CONTENT_TYPE_JSONLD);
```

Comment:
I've asked Hugo a question whether we want to also align responses for Fulltext Write methods. The specs say to also add the UTF-8 encoding, but don't say anything about whether or not to include IIIF version output. If so, I propose to reuse again the AcceptUtils.addContentTypeToResponseHeader() method

Reply:
Hugo says he wants to align these methods as well, although for Write methods it's a nice-to-have

Reply:
done